src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -28,16 +28,19 @@ struct ParsedOptions {
     db: Option<String>,
 }
 
-// TODO RUST-300: re-enable these tests once async SRV resolution is implemented.
-// #[cfg_attr(feature = "tokio-runtime", tokio::test(core_threads = 2))]
-// #[cfg_attr(feature = "async-std-runtime", async_std::test)]
-#[allow(dead_code)]
+#[cfg_attr(feature = "tokio-runtime", tokio::test(core_threads = 2))]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn run() {
     let run_test = |mut test_file: TestFile| {
+        // TODO DRIVERS-796: unskip this test
+        if test_file.uri == "mongodb+srv://test5.test.build.10gen.cc/?authSource=otherDB" {
```

Comment:
Does the test have a name we can use to skip instead of hard-coding the URI?

Reply:
Unfortunately not. The seedlist discovery tests only include a "comment" string for tests that are supposed to error. We could skip based on test file name, but that would require changes to the spec test runner function.

Reply:
I see. This is better than skipping based on the file name.